Sources/SnapshottingTests/PreviewTest.m

```diff
@@ -48,6 +48,10 @@ - (XCUIApplication *)getApp {
   return NULL;
 }
 
+- (BOOL)handleSnapshotPreviewsAsRegex {
```

Comment:
Do we need this? If we do can we make it default to YES and have it apply to both the include and exclude functions? I think it would be confusing if they behaved differently

Reply:
Yeah I think you're right. I did this initially because of what I wrote in the description (during our call on monday you called out that it should be backwards compatible), but ya you're right, lets just make them both regex by default